lemminx-liberty/src/test/java/io/openliberty/LibertyDiagnosticTest.java

```diff
@@ -718,6 +718,22 @@ public void testUniquenessForNameChangedFeatureDiagnostic() throws BadLocationEx
         XMLAssert.testDiagnosticsFor(serverXML, null, null, serverXMLURI,
                 invalid);
 
+        serverXML = String.join(newLine, //
+                "<server description=\"Sample Liberty server\">", //
+                "       <featureManager>", //
+                "               <platform>javaee-7.0</platform>", //
+                "               <feature>enterpriseBeans-4.0</feature>", //
+                "               <feature>ejb-3.2</feature>", //
+                "       </featureManager>", //
+
+                "</server>" //
+        );
+        invalid = new Diagnostic();
+        invalid.setRange(r(4, 24, 4, 31));
+        invalid.setMessage("ERROR: The ejb-3.2 feature cannot be configured with the enterprisebeans-4.0 feature because they are two different versions of the same feature. The feature name changed from ejb to enterprisebeans for Jakarta EE. Remove one of the features.");
```

Comment:
I'm glad I had you add this test. The feature name `enterprisebeans-4.0` should be `enterpriseBeans-4.0`. It is not ok for our message to have the wrong case for the feature. This means your code needs to change to be able to use the specified feature name, not the lowercase version of it.

Reply:
Done